azkaban-webserver/src/web/js/azkaban/view/schedule-panel.js

```diff
@@ -39,40 +37,32 @@ azkaban.SchedulePanelView = Backbone.View.extend({
   },
 
   scheduleFlow: function() {
-    var timeVal = $('#timepicker').val();
     var timezoneVal = $('#timezone').val();
-
-    var dateVal = $('#datepicker').val();
-
-    var is_recurringVal = $('#is_recurring').val();
-    var periodVal = $('#period').val();
-    var periodUnits = $('#period_units').val();
-
     var scheduleURL = contextURL + "/schedule"
     var scheduleData = flowExecuteDialogView.getExecutionOptionData();
 
-    console.log("Creating schedule for " + projectName + "." +
-        scheduleData.flow);
+    console.log("Creating schedule for " + projectName + "." + scheduleData.flow);
 
-    var scheduleTime = moment(timeVal, 'h/mm A').format('h,mm,A,') + timezoneVal;
-    console.log(scheduleTime);
+    var currentMomentTime = moment();
```

Comment:
Does it return the time based on the browser's time zone which may be different from the server's time zone?